install/prerequisites/thunder/generate-thunder-secrets.sh

```
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+source "${SCRIPT_DIR}/.config.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./.config.sh was not specified as input (see shellcheck -x).

(SC1091)